From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Sat, 5 Jul 2025 23:50:13 -0500
Subject: [PATCH] Fix build with slibtool

Fixes the following build error
│B│   CCLD     libddcutil.la
│B│   CCLD     ddcutil
│B│ clang: error: unknown argument: '--disable-static'

---
 src/Makefile.am | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 646ef411..3a67e92b 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -312,10 +312,6 @@ if ASAN_COND
    libddcutil_la_LIBADD += -lasan 
 endif
 
-# doesn't prevent creation of .la 
-# try disabling to create libddcutil.a - doesnt do it
-libddcutil_la_LDFLAGS += --disable-static
-
 endif
 
 all-local:
